### PR TITLE
build: add Kokoro configs

### PR DESCRIPTION
**Pull request by jkwlui**

### CODE REVIEW

File: synth.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Google LLC
```
Comment (ace-n) [inline synth.py]: This is a bit confusing if it's in the root of the repo.

Do we even need it, given that this isn't an autogenerated repo?

File: .kokoro/common.cfg [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```
Comment (ace-n) [inline .kokoro/common.cfg]: Nit: will the `.kokoro` file be included in `npm` installs of this library?

(Ideally it wouldn't be, but not the end of the world if it is.)

Comment (jkwlui) [reply .kokoro/common.cfg]: I added `files` field in package.json, so only the source will be included.

File: synth.py (hunk 1 above)
Comment (jkwlui) [reply synth.py]: removed

**Pull request merged**